node/npm_dependencies.ts

```diff
@@ -267,6 +268,10 @@ export const vendorNPMSpecifiers = async ({
     splitting: true,
     target: 'es2020',
     write: false,
+    define: {
+      // unsure if this is a wise setting, but it seems to work for this case. should we set "development" for local CLI?
+      'process.env.NODE_ENV': '"production"',
```

Comment:
Because we're bundling a Node module to become a Deno module, no Node globals are available. Deno's native NPM support supplies these by default, we'll have to work around this. A couple thoughts:

1. if we can find a way to tell Deno to treat this file as a Node module, that'd be ideal
1. In local dev, maybe a polyfill that populates globalThis.process with Deno's node:process would work better here
1. In builds, I think it's good to define this during bundling, so we don't bundle too much code in. But we should probably not hard-code it.